tao_compiler/mlir/xla/ral/ral_context.cc

```diff
@@ -186,7 +186,7 @@ void Context::signalErrorLocked(status_t errcode, const std::string& err_msg) {
   Impl::Status st;
   st.errcode = errcode;
   st.err_msg = err_msg;
-  TAO_VLOG(0) << "[[ERROR]] Context catch an exception: " << err_msg;
+  TAO_LOG(FATAL) << "[[ERROR]] Context catch an exception: " << err_msg;
```

Comment:
As I could remember, errors are expected to get handled outside RAL.

Reply:
Most of the errors are usually not recoverable (e.g. LLVM JIT code is more like c, with no cleanup like exception in C++).